deskpro/deskpro-admin-frontend
# Adds Cypress

To test:
- `npm install`
- `npx cypress open`
- In interactive editor, select Table test
- See test run and complete.
